File: python/agentic_commerce_protocol/models/fulfillment_option_shipping.py
# this file is @generated
import typing as t

from .common import BaseModel


class FulfillmentOptionShipping(BaseModel):
    id: str
    title: str
    subtitle: t.Optional[str] = None
    carrier: t.Optional[str] = None
    earliest_delivery_time: t.Optional[str] = None
    latest_delivery_time: t.Optional[str] = None
    subtotal: int
    tax: int
    total: int

File: kotlin/src/main/kotlin/com/svix/agentic_commerce_protocol/FulfillmentOptionShipping.kt
// This file is @generated
package com.svix.agentic_commerce_protocol

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class FulfillmentOptionShipping(
    val id: String,
    val title: String,
    val subtitle: String? = null,
    val carrier: String? = null,
    @SerialName("earliest_delivery_time") val earliestDeliveryTime: String? = null,
    @SerialName("latest_delivery_time") val latestDeliveryTime: String? = null,
    val subtotal: Long,
    val tax: Long,
    val total: Long,
)
